zetaclient/chains/bitcoin/fee.go

```diff
@@ -15,27 +15,41 @@ import (
 	clientcommon "github.com/zeta-chain/zetacore/zetaclient/common"
 )
 
 const (
-	bytesPerKB              = 1000
-	bytesPerInput           = 41  // each input is 41 bytes
-	bytesPerOutputP2TR      = 43  // each P2TR output is 43 bytes
-	bytesPerOutputP2WSH     = 43  // each P2WSH output is 43 bytes
-	bytesPerOutputP2WPKH    = 31  // each P2WPKH output is 31 bytes
-	bytesPerOutputP2SH      = 32  // each P2SH output is 32 bytes
-	bytesPerOutputP2PKH     = 34  // each P2PKH output is 34 bytes
-	bytesPerOutputAvg       = 37  // average size of all above types of outputs (36.6 bytes)
-	bytes1stWitness         = 110 // the 1st witness incurs about 110 bytes and it may vary
-	bytesPerWitness         = 108 // each additional witness incurs about 108 bytes and it may vary
-	defaultDepositorFeeRate = 20  // 20 sat/byte is the default depositor fee rate
-
-	OutboundBytesMin = uint64(239)  // 239vB == EstimateSegWitTxSize(2, 2, toP2WPKH)
-	OutboundBytesMax = uint64(1543) // 1543v == EstimateSegWitTxSize(21, 2, toP2TR)
-	OutboundBytesAvg = uint64(245)  // 245vB is a suggested gas limit for zetacore
-
-	DynamicDepositorFeeHeight = 834500 // DynamicDepositorFeeHeight contains the starting height (Bitcoin mainnet) from which dynamic depositor fee will take effect
+	// constants related to transaction size calculations
+	bytesPerKB           = 1000
+	bytesPerInput        = 41           // each input is 41 bytes
+	bytesPerOutputP2TR   = 43           // each P2TR output is 43 bytes
+	bytesPerOutputP2WSH  = 43           // each P2WSH output is 43 bytes
+	bytesPerOutputP2WPKH = 31           // each P2WPKH output is 31 bytes
+	bytesPerOutputP2SH   = 32           // each P2SH output is 32 bytes
+	bytesPerOutputP2PKH  = 34           // each P2PKH output is 34 bytes
+	bytesPerOutputAvg    = 37           // average size of all above types of outputs (36.6 bytes)
+	bytes1stWitness      = 110          // the 1st witness incurs about 110 bytes and it may vary
+	bytesPerWitness      = 108          // each additional witness incurs about 108 bytes and it may vary
+	OutboundBytesMin     = uint64(239)  // 239vB == EstimateSegWitTxSize(2, 2, toP2WPKH)
+	OutboundBytesMax     = uint64(1543) // 1543v == EstimateSegWitTxSize(21, 2, toP2TR)
+	OutboundBytesAvg     = uint64(245)  // 245vB is a suggested gas limit for zetacore
+
+	// defaultDepositorFeeRate is the default fee rate for depositor fee, 20 sat/vB
+	defaultDepositorFeeRate = 20
+
+	// defaultTestnetFeeRate is the default fee rate for testnet, 10 sat/vB
+	defaultTestnetFeeRate = 10
+
+	// feeRateCountBackBlocks is the default number of blocks to look back for fee rate estimation
+	feeRateCountBackBlocks = 2
+
+	// DynamicDepositorFeeHeight is the mainnet height from which dynamic depositor fee V1 is applied
+	DynamicDepositorFeeHeight = 834500
+
+	// DynamicDepositorFeeHeightV2 is the mainnet height from which dynamic depositor fee V2 is applied
+	DynamicDepositorFeeHeightV2 = 863400
```

Comment:
Can we explain in comment why this height is used

Reply:
updated comment: https://github.com/zeta-chain/node/pull/2765/commits/3780897e91245faf820dab0c3efd404ab92ad8bf